app.js

```diff
@@ -45,7 +45,7 @@ async function unzip({
   await validatePaths(absoluteArchivePath);
 
   const destinationPathExists = await pathExists(absoluteDestinationPath);
-  if (!clearExtractionPath && destinationPathExists) {
+  if (!clearExtractionPath && destinationPathExists && destinationPath !== ".") {
```

Comment:
I've talked to @kenskyfish about this. We should allow the user to extract multiple archives into the same directory, no matter if it's CWD. This condition prevents that.